lib/rubocop/target_ruby.rb

```diff
@@ -99,7 +99,7 @@ def version_from_gemspec_file(file)
         processed_source = ProcessedSource.from_file(
           file, DEFAULT_VERSION, parser_engine: @config.parser_engine
         )
-        required_ruby_version(processed_source.ast).first
+        required_ruby_version(processed_source.ast).first if processed_source.valid_syntax?
```

Comment:
Can you tweak to the following?

```suggestion
        return unless processed_source.valid_syntax?

        required_ruby_version(processed_source.ast).first 
```